### PR TITLE
Disable popup if in PrivateBrowsing

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

**Comment by bakulf** [comment]
This is about issue #123

**Comment by jonathanKingston** [comment]
@bakulf does this work in 'always private mode'?
Recent events in a certain country has lead me to suggest everyone should use this mode.

File: webextension/background.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+browser.tabs.onCreated.addListener(tab => {
+  if (tab.incognito) {
+    browser.browserAction.disable(tab.id);
```
Comment (groovecoder) [inline webextension/background.js]: let's add `browser.browserAction.setTitle("Containers disabled in Private Browsing Mode")` so we can show the user what's happening via the tooltip.

**Comment by groovecoder** [comment]
And yes - I verified that this disables the button for users who are in "always Private Browsing" mode. (I.e., "Never remember history" in Firefox settings).

nightly-always-private

**Comment by jonathanKingston** [comment]
(didn't finish posting this comment until now)
@groovecoder thats a shame :(.
Hopefully the change to enable in that mode won't be too complex. We discussed this for central and said we would do this. 
- I will try and fix in a follow up

**Comment by bakulf** [comment]
PR #133 is about it.

**Pull request merged**